src/parse.js

```diff
@@ -12,8 +12,22 @@ const STATE_MODIFIER = Symbol("modifier");
 const STATE_FUNCTION = Symbol("function");
 const STATE_NAME = Symbol("name");
 
-export function parseCell(input, {globals} = {}) {
-  const cell = CellParser.parse(input);
+
+export function parseCell(input, {globals, tag, raw} = {}) {
+  return tag != null
+    ? parseTemplateCell(input, {globals, tag, raw})
+    : parseJavaScriptCell(input, {globals});
+}
+
+function parseJavaScriptCell(input, options) {
+  return finishCell(CellParser.parse(input), input, options);
```

Comment:
We’ll want to pass through any acorn options here, too:

```suggestion
  return finishCell(CellParser.parse(input, options), input, options);
```